test/core/consensus/beefy_test.cpp

```diff
@@ -132,6 +134,8 @@ struct BeefyTest : testing::Test {
       return genesisVoters();
     });
     EXPECT_CALL(*timeline_, wasSynchronized()).WillRepeatedly(Return(true));
+    EXPECT_CALL(*synchronizer, fetchHeadersBack(_, _, _))
+        .WillRepeatedly(Return(true));
```

Comment:
Test doesn't call `fetchHeadersBack`, because no blocks are missing

Reply:
Test build fails without it